src/main/java/frc/robot/utils/Alignable.java

```diff
@@ -4,7 +4,7 @@
 
 public enum Alignable {
     AMP(0,0,0,0),
-    SPEAKER(-0.0381+0.234809,16.46,5.548,2.00);//1.343 - dist center of robot to speaker shot //value v2 1.343 - 0.447649 - Constants.ROBOT_LENGTH / 2
+    SPEAKER(-0.0381+0.234809,16.58-0.234809,5.548,2);//1.343 - dist center of robot to speaker shot //value v2 1.343 - 0.447649 - Constants.ROBOT_LENGTH / 2
```

Comment:
Is there a reason to leave these as math operations instead of just doing the math?

Reply:
I was not confident on the right side of the equations. But the right side is known measured offset